main.py

```diff
@@ -1,10 +1,45 @@
+import json
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_info = json.load(file)
```

Comment:
The context manager is needed to work with the file, in our case, to read data. Therefore, after you have read the data from the file, exit the block.